### DISCUSSION TITLE
CommitChange API

### DISCUSSION THREAD

**Discussion by yoan34** [houdiniproject/houdini#947] [General]
Hi everyone,

I am a work-study developer for a startup called Jokosun, we are looking for a platform to help us with an investment system.

I am unfamiliar with ruby ​​and am looking to see if an api exists to retrieve essential data from CommitChange. I noticed that the url **localhost:5000/api/v1/nonprofit/1** is working. I can't find any other urls.

do they exist? or should we add them to a file?

thanks in advance

**Comment by wwahammy** [comment]
Hey @yoan34, welcome to Houdini! This wouldn't be the place to ask questions about CommitChange, Houdini is an independent project. CommitChange uses it and contributes back to Houdini but Houdini is independent.

If you're using Houdini, there are a number of APIs which you can to get data for a nonprofit. They're not well documented but they're listed in the `config/routes.rb` file inside `namespace :api`. If you'd like to document the api more, that would be much appreciated.

**Reply by yoan34** [reply]
Hi, thank you for your reply.
Indeed in this file there are the routes. I am using the 1-0 stable version to use it with docker, the main branch, I cannot launch it.

Glad to see the API is more elaborate.
